api/envoy/config/filter/http/buffer/v2/buffer.proto

```diff
@@ -19,13 +19,9 @@ message Buffer {
 
   // The maximum number of seconds that the filter will wait for a complete
   // request before returning a 408 response.
-  google.protobuf.Duration max_request_time = 2 [
-    (validate.rules).duration = {
-      required: true,
-      gt: {}
-    },
-    (gogoproto.stdduration) = true
-  ];
+  // This is deprecated in favour of http conn manager request timeouts.
+  google.protobuf.Duration max_request_time = 2
```

Comment:
Can you check the buffer filter code to make sure this won't crash now that it isn't required?